src/pages/home/report/ReportActionsView.js

```diff
@@ -200,9 +194,9 @@ class ReportActionsView extends React.Component {
         const didReportBecomeVisible = isReportFullyVisible && (didSidebarClose || didScreenSizeIncrease);
         if (didReportBecomeVisible) {
             this.setState({
-                newMarkerSequenceNumber: !ReportUtils.isUnread(this.props.report)
-                    ? 0
-                    : this.props.report.lastReadSequenceNumber + 1,
+                newMarkerReportActionID: !ReportUtils.isUnread(this.props.report)
+                    ? null
+                    : this.computeNewMarkerReportActionID(),
```

Comment:
Another alternative would be to just sort the reportActions again in `computeNewMarkerReportActionID`, but it would come at a performance cost because we would be re-sorting an already-sorted array.

I think I'd prefer to just do your suggestion above and make it more clear that the function currently called `computeNewMarkerReportActionID` needs a sorted array of reportActions

Reply:
Moved to `ReportUtils` as discussed.